### DISCUSSION TITLE
Broken "See this query" link in GitHub Code Search Beta banner

### DISCUSSION THREAD

**Discussion by thibaudcolas** [community/community#34108] [Other Feature Feedback, Questions, & Ideas]
If I follow the banner’s "see this query" link on the GitHub search results page, I arrive onto cs.github.com _without_ the language filter currently in use.

Example page: https://github.com/django/djangoproject.com/search?l=html

Here is what the banner looks like for me:

<img width="744" alt="code search beta banner" src="https://user-images.githubusercontent.com/877585/192135171-9e32ef3b-9b9f-4c85-8684-90aaa15a7ee5.png">

And here is the HTML for the link:

```
<a rel="noopener noreferrer" target="_blank" href="https://cs.github.com/django/djangoproject.com?q=">See this query</a>
```

**Comment by colinwm** [comment]
Good observation! We'll fix this. Thanks for pointing it out
